core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/economy/migrate/MigrationEconomy.java

```diff
@@ -69,29 +75,20 @@ public boolean isPrimary() {
             }
 
             @Override
-            public boolean supportsNegativeBalances() {
-                return false; // todo: ?????
+            public TriState supportsNegativeBalances() {
+                return TriState.UNSPECIFIED;
```

Comment:
What's wrong with returning a value indicating that the migration economy doesn't support negative balances? It doesn't support balances at all, but that's beside the point. Changing the entire API for what is just an ideally very temporary placeholder that denies access feels weird.

Reply:
we're in the 2.0.0 branch. supportsNegativeBalances method was added in 2.0.0 . we can break it as we wish. what's wrong with returning TrIState rather than a boolean?

Reply:
I don't understand why it needs to be an ambiguous value at all. The migration economy does not support negative balances. Granted, it also does not support balances, but that's not the point really - there is not negative balance support, so a return value of false is accurate.